.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Hypothesis Google Forum
+    url: /https://groups.google.com/a/list.hypothes.is/g/dev
```

Comment:
I suggest we link to the "Contact Hypothesis" page here instead: https://web.hypothes.is/contact/. This will ensure that it stays up to date with the support team's preferred practices as well as the various communication channels that are available (Slack). Regarding Google Groups, we have two forums, one is a developer-focused forum and the other is [for end users](https://groups.google.com/g/hypothesis-forum). The one for end users is probably more appropriate here and is also monitored by the support team.